release-kong.sh

```diff
@@ -10,14 +10,14 @@ PULP_USERNAME=
 PULP_PASSWORD=
 
 # release finals into prod, others into stage
-if [[ "$KONG_VERSION" =~ ^[0-9]+\.[0-9]+\.[0-9]+(\.[0-9]+)?$ ]]; then
```

Comment:
`KONG_VERSION` looks really overloaded. Right now my tag is `3.0.0.0-tyler` but `KONG_VERSION` is read from the ee files so comes in as `3.0.0.0` and fell into the 'prod' path of this tree.

Possibilities:
Depend more on `OFFICIAL_RELEASE` and a new `KONG_TAG` environment variable
Set a suffix in kong/enterprise_edition/meta.lua (I think this is how CE handles this)

Reply:
I can definitely see us coming back to address this later. Right now commits to `master` and `next/3.0.x.x` go to pulp prod, but if I tag `3.0.0.0-preview` from `next/3.0.x.x` it will go to pulp dev